### PR TITLE
docs/variants/msi_z690/openness_analysis.md: add openness score

### PR DESCRIPTION
**Pull request by miczyg1**
Signed-off-by: Michał Żygowski <michal.zygowski@3mdeb.com>

### CODE REVIEW

File: docs/variants/msi_z690/openness_analysis.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
+# Intro
+
+This document aims to compare the openness of Dasharo firmware and AMI BIOS.
```
Comment (pietrushnic) [inline docs/variants/msi_z690/openness_analysis.md:3]:
```suggestion
This document aims to compare the openness of Dasharo firmware and AMI BIOS for MSI PRO Z690-A DDR4 WiFI.
```

File: docs/variants/msi_z690/openness_analysis.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+There is an
+[ongoing discussion](https://github.com/Dasharo/dasharo-issues/issues/43)
+about the methodology of the openness metric.
+
+## BIOS versions used in the analysis
+
+* [Dasharo v1.0.0](https://3mdeb.com/open-source-firmware/Dasharo/msi_ms7d25/v1.0.0/msi_ms7d25_v1.0.0.rom)
```
Comment (pietrushnic) [inline docs/variants/msi_z690/openness_analysis.md:11]: 404

Comment (miczyg1) [reply docs/variants/msi_z690/openness_analysis.md:11]: Yes, because it is not yet published.

Comment (miczyg1) [reply docs/variants/msi_z690/openness_analysis.md:12]: Should be fixed already.

**Comment by miczyg1** [comment]
@pietrushnic any more sugestions?

**Review by macpijan — APPROVED** [review]

**Pull request merged**